src/applications/claims-status/tests/e2e/00.claims-list-breadcrumb.cypress.spec.js

```diff
@@ -0,0 +1,58 @@
+const Timeouts = require('platform/testing/e2e/timeouts.js');
+
+import claimsList from './fixtures/mocks/claims-list.json';
+
+describe('Breadcrumb Test', () => {
+  it('Verifies breadcrumb functionality', () => {
+    cy.intercept('GET', '/v0/evss_claims_async', claimsList);
+    cy.visit('/track-claims');
+    cy.login();
+    cy.title().should('eq', 'Track Claims: VA.gov');
+    cy.get('.claim-list-item-container', { timeout: Timeouts.slow }).should(
+      'be.visible',
+    );
+    cy.get('.va-nav-breadcrumbs', { timeout: Timeouts.normal }).should(
+      'be.visible',
+    );
+    cy.get('.va-nav-breadcrumbs-list', { timeout: Timeouts.normal }).should(
+      'be.visible',
+    );
+    cy.get('a[aria-current="page"', { timeout: Timeouts.normal }).should(
+      'be.visible',
+    );
+    cy.injectAxeThenAxeCheck();
+
+    cy.get(
+      '.va-nav-breadcrumbs-list li:nth-of-type(2) a[aria-current="page"]',
+    ).should('exist');
+    cy.get(
+      '.va-nav-breadcrumbs-list li:nth-of-type(2) a[aria-current="page"]',
+    ).should('contain', 'Check your claims and appeals');
+    cy.get(
+      '.va-nav-breadcrumbs-list li:nth-of-type(2) a[aria-current="page"]',
+    ).should('have.css', 'pointer-events', 'none');
+
+    cy.viewport(375, 667);
```

Comment:
Is this a mobile viewport? We are trying to move away from using hardcoded viewports and using the most common viewports instead with `cy.viewportPreset()`. See https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/testing/end-to-end/cypress-best-practices-on-vsp.md#viewport-presets-cyviewportpresetpreset-orientation-options

Reply:
Didn't know about these, but what a great idea to keep things up to date!  Went ahead and converted this to top mobile viewport #1